### PR TITLE
Add multi volgroup management when using autodiscovery option

### PR DESCRIPTION
**Pull request by ashangit**

### CODE REVIEW

File: bin/check-fs-writable.rb [hunk 1]
```diff
@@ -66,10 +66,23 @@ def usage_summary
...
   def acquire_mnt_pts
-    `grep VolGroup /proc/self/mounts | awk '{print $2, $4}' | awk -F, '{print $1}' | awk '{print $1, $2}'`
+    mnt_pts = []
+    vol_groups = acquire_vol_groups.split("\n")
+    vol_groups.each  do |vol_group|
+      `grep #{vol_group} /proc/self/mounts | awk '{print $2, $4}' | awk -F, '{print $1}' | awk '{print $1, $2}'`.split("\n").each do |mnt|
+        mnt_pts << mnt
+      end
+    end
+    return mnt_pts
```
Comment (tas50) [inline bin/check-fs-writable.rb]: This can just be mnt_pts.  No need for the return

File: bin/check-fs-writable.rb [hunk 2]
```diff
@@ -66,10 +66,23 @@ def usage_summary
     end
   end
 
+  # Get the volgroups
+    #
+    def acquire_vol_groups
```
Comment (tas50) [inline bin/check-fs-writable.rb]: The indentation on this method doesn't match the other methods

File: bin/check-fs-writable.rb [hunk 3]
```diff
@@ -113,6 +124,7 @@ def auto_discover
     rw_test?(mount_info)
     puts "The critical mount points according to proc are: #{ @crit_pt_proc }" if config[:debug]
     puts "The critical mount points according to actual testing are: #{ @crit_pt_test }" if config[:debug]
+    return true
```
Comment (tas50) [inline bin/check-fs-writable.rb]: This can just be true

**Comment by mattyjones** [comment]
@ashangit  Thanks for taking care of this, it has been on my shit to not get done list for a while.!

Can you make the changes @tas50  suggested and then we can look at the tRavis output and get this built and pushed.

**Comment by ashangit** [comment]
@mattyjones I have made all the change that @tas50 has suggested. Let me know if I need to update something else.

**Comment by mattyjones** [comment]
@tas50  Is this crap what you ment by the rubocop changes?  I have the gemspec pinned at 0.30.

I always put spaces in so this is going to bite hard.

**Comment by tas50** [comment]
@mattyjones Travis brought in 0.32 for that build

**Comment by mattyjones** [comment]
@tas50  @ashangit 

I am going to merge this anyway and deal with the rubocop followout

**Pull request merged**